assets/sass/components/global/_googlesitekit-entity-search.scss

```diff
@@ -40,9 +39,16 @@
 		}
 
 		.googlesitekit-entity-search__close {
+			border-radius: 50%;
 			height: 32px;
-			min-width: 36px;
+			min-height: 36px;
+			min-width: 36px !important;
```

Comment:
@hussain-t Let's avoid using `!important` unless we don't have any options. For e.g if there are inline styles. What's happening here is that `min-width` is getting overridden by a the `min-width` of a more specific selector, which is: `.googlesitekit-plugin .googlesitekit-entity-search .mdc-button`. So in this case, we can look into using the existing selector, or add more specificity to `.googlesitekit-plugin .googlesitekit-entity-search .googlesitekit-entity-search__close` by doing something like that: `.googlesitekit-plugin .googlesitekit-entity-search .mdc-button.googlesitekit-entity-search__close`